openapi/data_repository_service.swagger.yaml

```diff
@@ -282,20 +336,53 @@ definitions:
     properties:
       object:
         $ref: '#/definitions/Object'
-  URL:
+  GetAccessURLResponse:
```

Comment:
Does this still need to be an object? Looks like it's now just a string, which would simplify syntax. (I'm open to others input here, since I can imagine that we'll end up adding some other properties down the road.)

Reply:
All responses so far have been an object, so I kept it that way. There might be some cases where redirection may not be possible as a response and/or we might add more fields in the future as you mentioned (expiry date etc).

Reply:
Sounds good -- I'm happy leaving it an object for consistency and future-proofing. Resolving.